mozilla-release/browser/app/profile/firefox.js

```diff
@@ -277,16 +277,8 @@ pref("browser.warnOnQuit", true);
 pref("browser.fullscreen.autohide", true);
 pref("browser.overlink-delay", 80);
 
-#ifdef UNIX_BUT_NOT_MAC
```

Comment:
Lets not delete the lines, to avoid future merge conflicts.
We can wrap this in `#if 0` and below that define the prefs
Adding a comment will also help to avoid this in future merges.

Reply:
I force-pushed to fix the commit with your suggestion. I did not yet try to compile locally (will do so now). Is it correct to assume that the preprocessor to evaluate these `#if` blocks support nested conditions?